server/settings/tools.py

```diff
@@ -448,6 +448,17 @@ class GlobalToolsModel(BaseSettingsModel):
                 "task_types": [],
                 "tasks": [],
                 "template": "SK_{folder[name]}{variant}"
+            },
+            {
+                "product_types": [
+                    "hda"
+                ],
+                "hosts": [
+                    "houdini"
+                ],
+                "task_types": [],
+                "tasks": [],
+                "template": "{folder[label]}_{variant}"
```

Comment:
Is it really a good idea to use the folder label in a product name template since it's allowed for a folder label to have characters that are not allowed on a product name, like brackets or whatever.

Reply:
Oh, I thought the folder label is equivalent to `folder_path.split("/")[-1]`.

Reply:
![image](https://github.com/ynput/ayon-core/assets/33513211/105927ae-18b0-46e0-b550-59176469ad3e)

Reply:
This would have to be `folder["name"]` to just get `sh010`
If you want the full unique folder path then: `folder["path"]` but note that that includes the `/` like `/shots/sh010`.

Reply:
> ![image](https://private-user-images.githubusercontent.com/33513211/339365014-105927ae-18b0-46e0-b550-59176469ad3e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTgzMDExOTcsIm5iZiI6MTcxODMwMDg5NywicGF0aCI6Ii8zMzUxMzIxMS8zMzkzNjUwMTQtMTA1OTI3YWUtMThiMC00NmUwLWI1NTAtNTkxNzY0NjlhZDNlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjEzVDE3NDgxN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTM3ODY3ZjFkYjk1NTdjMzk4MjhlZWY5MGY0YTZhOGNmMDQzYmUxNTM4MzFmMGFjMTZjNzUzNjMyZTUxODQ0ZTEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.mI14gfndAHRUiS9NjrwH9JLdsT2l9H2AHvgQ7E8taqw)

Omg, what is that! 
let me change it to `folder[name]`